store/options.go

```diff
@@ -27,6 +32,7 @@ func DefaultParameters() Parameters {
 		StoreCacheSize: 4096,
 		IndexCacheSize: 16384,
 		WriteBatchSize: 2048,
+		StorePrefix:    datastore.NewKey("headers"),
```

Comment:
I would propose keeping the default value empty and configuring the store prefix explicitly for all entities you'll need

Reply:
node can adapt changes and use the option. Keeping in mind that this is a lib and could be re-used by any other project, having` datastore.NewKey("headers")` as default looks a bit strange to me. But I do not insist

Reply:
We store headers so it's fine to name it headers. If we find a better name for this datastracuture, we will change the default as well.